src/filters/multipart.rs

```diff
@@ -207,3 +208,15 @@ impl Stream for BodyIoError {
         }
     }
 }
+
+/// An error used when a multipart field is missing a name.
+#[derive(Debug)]
+pub struct MultipartFieldMissingName;
```

Comment:
```suggestion
struct MultipartFieldMissingName;
```

I'd keep it private. _Or_, make it have a private field, so we could extend it latter if need be.